src/messageHandlers.ts

```diff
@@ -150,6 +150,7 @@ const messageHandlers:Record<IpcMessageType, MessageHandler> = {
 
 		const newNote = {
 			...note,
+			id: '',
 			title: note.title + ' - Copy',
 			tags: tagTitles.join(','),
 		};
```

Comment:
I'm suggesting delete the ID instead but otherwise looks good. Thanks for fixing this!

```suggestion
		const newNote = {
			...note,
			title: note.title + ' - Copy',
			tags: tagTitles.join(','),
		};
		
		delete newNote.id;
```